### ISSUE TITLE
command line tools for what we normally do in iraf

### ISSUE THREAD

**Issue by crawfordsm** [astropy/ccdproc#207]
[ ] imhead

**Comment by mwcraig** [comment]
Could this be an alias to fitsheader (already included in astropy core)? Or maybe a table in the docs like:

To do this IRAF task... |  ...do this from ccdproc (or astropy)
------------------------------|----------------------------------
imhead  | fitsheader

**Comment by crawfordsm** [comment]
Yup, it could, I was just adding to here things that came up during writing the tutorial
